tests/test_bridge_domain.rb

```diff
@@ -43,11 +43,11 @@ def teardown
   def test_single_bd_create_destroy
     bd = BridgeDomain.new('100', true)
     bds = BridgeDomain.bds
-    assert(bds.key?(100), 'Error: failed to create bridge-domain 100')
+    assert(bds.key?(100.to_s), 'Error: failed to create bridge-domain 100')
```

Comment:
Wanted to keep every return value consistent with other getters as all return in boolean or string formats.